projects/opentrain/israel_railways_pipeline/datahandler/DataHandler.java

```diff
@@ -0,0 +1,241 @@
+                                .setValid(record.get("valid"))
+                                .setInvalidReason(record.get("invalid_reason"))
+                                .setXWeekDayLocal(record.get("x_week_day_local"))
+                                .setXHourLocal(record.get("x_hour_local"))
+                                .setRouteId(record.get("route_id"))
+                                .setXAvgDelayArrival(record.get("x_avg_delay_arrival"))
+                                .setXCacheVersion(record.get("x_cache_version"))
+                                .setXMax2DelayArrival(record.get("x_max2_delay_arrival"))
+                                .setXBeforeLastDelayArrival(record.get("x_before_last_delay_arrival"))
+                                .setXLastDelayArrival(record.get("setXLastDelayArrival"))   //error
+                                .build());
+            }
+            return tripsList;
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+
+
+    public static List<RailWayProtos.Routes> getRoutes(Path routeCsv) {
+
+        List<RailWayProtos.Routes> routesList = new ArrayList<>();
+        try {
+            CSVParser parser =
+                    CSVParser.parse(
+                            routeCsv,
+                            StandardCharsets.UTF_8,
+                            CSVFormat.DEFAULT.withHeader("id", "stop_ids"
+                            ));
+            boolean isHeader = true;
+            for (CSVRecord record : parser) {
+                if (isHeader) {
+                    isHeader = false;
+                    continue;
+                }
+
+                RailWayProtos.Routes.Builder routeBuilder = RailWayProtos.Routes.newBuilder();
+                routesList.add(
+                        routeBuilder
+                                .setId(Integer.parseInt(record.get("id")))
+                                .setStopIds(record.get("stop_ids"))
+                                .build());
+            }
+            return routesList;
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+
+    public static void saveRoutes(Path path, String outputName) {
+        try (FileOutputStream output = new FileOutputStream(outputName)) {
+            for (RailWayProtos.Routes route : getRoutes(path)) {
+                route.writeDelimitedTo(output);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+
+//
+//  // Paths to the CSVs files
+//  static Path routesAbsPath;
+//  // Paths to output files
+//  static String routesProtoAbsPath;
+
+    public static void main(String[] args) {
+//    {
+//      routesAbsPath = Paths.get(args[0]);
+//      routesProtoAbsPath = args[1];
+//    }
+//        getRoutes(Paths.get(args[0]));
+//        System.out.println(getRoutes(Paths.get(args[0])));
+        saveRoutes(Paths.get(args[0]), args[1]);
```

Comment:
It's input arguments:
```
0 - external/israel_railways_data/routes.csv
1 - bazel-out/k8-fastbuild/genfiles/projects/opentrain/israel_railways_pipeline/routes.protobin
2 - external/israel_railways_data/samples.csv
3 - bazel-out/k8-fastbuild/genfiles/projects/opentrain/israel_railways_pipeline/samples.protobin
4 - external/israel_railways_data/stops.csv
5 - bazel-out/k8-fastbuild/genfiles/projects/opentrain/israel_railways_pipeline/stops.protobin
6 - external/israel_railways_data/trips.csv
7 - bazel-out/k8-fastbuild/genfiles/projects/opentrain/israel_railways_pipeline/trips.protobin
```
Please check and set correctly it according to methods requirements.